pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -675,7 +677,7 @@ private static string LaunchDockerWorker(int i, CommandLineArgs commandLineArgs,
             testRunnerCmd = AddPassThroughArguments(commandLineArgs, testRunnerCmd);
             testRunnerCmd += $" workerport={workerPort}";
 
-            string dockerArgs = $"run --name {workerName} -it --rm -m {workerBytes}b -v {PathEx.GetDownloadsPath()}:c:\\downloads -v {pwizRoot}:c:\\pwiz {RunTests.DOCKER_IMAGE_NAME} \"{testRunnerCmd} workername={workerName}\" {dockerRunRedirect}";

```

Comment:
I don't think a new TestRunner argument is needed. You can change the workerName to include the TestRunner process ID:
```
var pid = Process.GetCurrentProcess().Id;
string workerName = bigWorker ? $"docker_big_worker_{pid}_{i}" : $"docker_worker_{pid}_{i}";
```
Theoretically the PID could be re-used from the last TestRunner but that seems very unlikely. But if that's a concern, could use a timestamp instead:
```
var timestamp = DateTime.UtcNow.ToShortTimeString();
string workerName = bigWorker ? $"docker_big_worker_{timestamp}_{i}" : $"docker_worker_{timestamp}_{i}";
```

Reply:
Simpler is better, I'll try this out.